src/game.c

```diff
@@ -66,7 +66,7 @@ Game *create_game(const char *level_folder,
     game->font = PUSH_LT(
         lt,
         create_sprite_font_from_file(
-            "images/charmap-oldschool.bmp",
+            "./assets/images/charmap-oldschool.bmp",
```

Comment:
Assets is the CWD (Current Working Directory) of the game, prefixing all of the hardcoded paths with `./assets/` breaks that assumptions.